cmd/fleetctl/get_test.go

```diff
@@ -2743,6 +2753,30 @@ func TestGetMDMCommands(t *testing.T) {
 +----+----------------------+---------------------------------------+--------------+----------+
 | u3 | 2023-04-11T09:05:00Z | InstallProfile                        |          200 | host2    |
 +----+----------------------+---------------------------------------+--------------+----------+
+`))
+
+	// Test with invalid option
+	_, err = runAppNoChecks([]string{"get", "mdm-commands", "--invalid", "foo"})
+	require.Error(t, err)
+
+	// Test with host identifier filter
+	listErr = nil
+	empty = false
+	expectIdentifier = true
+	_, err = runAppNoChecks([]string{"get", "mdm-commands", "--host-identifier", "foo"})
+	require.NoError(t, err)
+
+	// Test pagination flag
+	listErr = nil
+	empty = true
+	expectIdentifier = false
+	expectPage = true
+	res, err := runAppNoChecks([]string{"get", "mdm-commands", "--page", "1"})
```

Comment:
This won't be easy for a dev to use as is (we don't surface any more pages available etc)

Reply:
https://cli.github.com/manual/gh_gist_list

instead of --page I argued for -L --limit to be able to change the page size and --paginate to retrieve all pages but this was removed from the story and Noah never got back to how I could add the info into the figma (not an editor and don't think I need to be atm)

This way as a user I could specify -L 10 if I know I only need the latest 10 or if I need everything --paginate and have `fleetctl` iterate for me

Reply:
https://cli.github.com/manual/gh_api example w/ `--paginate`

Reply:
`gh gist list` is just an example of a cli w/ `-L, --limit` for a cli interfacing w/ pages

Reply:
cc: @noahtalerman

Reply:
Hey @georgekarrv, thanks for looping me in! We decided to cut paging through the results in `fleetctl` because it wasn't necessary for the problem we were trying to solve here: #19143

Every product change comes through a design review at Fleet. Anyone at Fleet can bring designs to design review. The designs don't have to be in Figma. They can be screenshots or, in this case, markdown in the issue.

Reply:
thx all, i'll pull out the existing pagination